metrics_utility/base/collector.py

```diff
@@ -273,7 +285,26 @@ def _gather_csv_collections(self):
          1) the temp file needs to be deleted to ensure enough disk space
          2) Collections with slicing function can produce duplicate filename
         """
+
+        # we will print the message every X seconds
+        seconds = 20
+        last_print_time = time.time()
+
+        self.logger.info('Starting collecting, this may take while.')
+
         for collection in self.collections[Collection.COLLECTION_TYPE_CSV]:
+            # print this message every X seconds
+            current_time = time.time()
+
+            if current_time - last_print_time >= seconds:
```

Comment:
hm, this is a fixed list of collections from the collections file? So no need to do any time checks? (this will jsut print ~1-7 lines based on number of collections allowed, or what does this print?)